kuma/users/jinja2/account/login.html

```diff
@@ -36,9 +36,15 @@ <h1>{{ _("Please sign in") }}</h1>
       {% endif %}
 
       {% if socialaccount_providers  %}
+        {% if settings.MULTI_AUTH_ENABLED %}
+          {% trans login_url=url('socialaccount_signin')|urlparams(next=request.get_full_path()) %}
```

Comment:
Likewise, you could simplify by using your new `login_url` function here:
```suggestion
          {% trans login_url=login_url() %}
```
which, like above, worked for me when I tested it locally.

Also, the text in line 33 above needs to change to include Google, so something like:
```jinja
      <p>{% trans %}
      In order to contribute, you need to sign in to MDN. You can sign in using
      GitHub or Google and a MDN profile will be created for you if you don't already
      have one.
      {% endtrans %}</p>
```

Reply:
No, because that copy won't matter any more. Since the new `settings.LOGIN_URL` means you don't end up this template at all (unless it's bookmarked or browser history). And that `settings.LOGIN_URL` is set depending on `settings.MULTI_AUTH_ENABLED`.